torchci/test/mergeBot.test.ts

```diff
@@ -316,4 +316,182 @@ describe("merge-bot", () => {
+            console.error("pending mocks: %j", scope.pendingMocks());
+        }
+        scope.done();
+    });
+
+    test("help using CLI", async () => {
+        const event = require("./fixtures/pull_request_comment.json");
+
+        event.payload.comment.body = `@pytorchbot help`;
+        const owner = event.payload.repository.owner.login;
+        const repo = event.payload.repository.name;
+        const pr_number = event.payload.issue.number;
+
+        const scope = nock("https://api.github.com")
+            .post(
+                `/repos/${owner}/${repo}/issues/${pr_number}/comments`,
+                (body) => {
+                    expect(JSON.stringify(body)).toContain(
+                        "To see all options for pytorchbot, " +
+                            "please refer to this [page](https://github.com/pytorch/pytorch/wiki/Bot-commands)."
+                    );
+                    return true;
+                }
+            )
+            .reply(200);
+
+        await probot.receive(event);
+        if (!scope.isDone()) {
+            console.error("pending mocks: %j", scope.pendingMocks());
+        }
+        scope.done();
+    });
+
+    test("revert using CLI", async () => {
+        const event = require("./fixtures/pull_request_comment.json");
+
+        event.payload.comment.body =
+            '@pytorchbot revert -m="test test test" -c="ghfirst"';
+
+        const owner = event.payload.repository.owner.login;
+        const repo = event.payload.repository.name;
+        const pr_number = event.payload.issue.number;
+        const comment_number = event.payload.comment.id;
+        const scope = nock("https://api.github.com")
+            .post(
+                `/repos/${owner}/${repo}/issues/comments/${comment_number}/reactions`,
+                (body) => {
+                    expect(JSON.stringify(body)).toContain('{"content":"+1"}');
+                    return true;
+                }
+            )
+            .reply(200, {})
+            .post(`/repos/${owner}/${repo}/dispatches`, (body) => {
+                expect(JSON.stringify(body)).toContain(
+                    `{"event_type":"try-revert","client_payload":{"pr_num":${pr_number},"comment_id":${comment_number}}}`
+                );
+                return true;
+            })
+            .reply(200, {});
+
+        await probot.receive(event);
+        if (!scope.isDone()) {
+            console.error("pending mocks: %j", scope.pendingMocks());
+        }
+        scope.done();
+    });
+
+    test("Random commands won't trigger CLI", async () => {
+        const eventCantMerge = require("./fixtures/pull_request_comment.json");
+        const eventWithQuotes = require("./fixtures/pull_request_comment.json");
+        const eventQuoted = require("./fixtures/pull_request_comment.json");
+        const testCommand = require("./fixtures/pull_request_comment.json");
+
+        eventCantMerge.payload.comment.body = "Can't merge closed PR #77376";
```

Comment:
This was the comment that was the issue and we added some additional comments that could potentially trip it up and validated that they won't trigger any events.